src/System.Private.CoreLib/shared/System/AggregateException.cs

```diff
@@ -451,6 +451,9 @@ public override string ToString()
 
             for (int i = 0; i < m_innerExceptions.Count; i++)
             {
+                if (m_innerExceptions[i] == InnerException)
```

Comment:
Is this fixing a regression introduced by the recent change to `Exception.ToString`?

Reply:
I don't think so. AggregateException constructors that take an `innerException` pass it to the base, and it becomes `InnerException`. They also put it into `m_innerExceptions`. Then in their ToString() they first start with `base.ToString()` that includes the `InnerException`, then they loop over their `m_innerExceptions` collection and emit those. This causes the original inner exception to be emitted twice.